application-templates/starter-typescript/src/test-utils/test-data/channel/transformers.ts

```diff
@@ -0,0 +1,41 @@
+import { Transformer, buildField } from '@commercetools-test-data/core';
+import type {
+  TChannel,
+  TChannelGraphql,
+} from '@commercetools-test-data/channel/dist/declarations/src/types';
+
+const transformers = {
+  default: Transformer<TChannel, TChannelGraphql>('default', {
+    buildFields: ['name', 'description', 'address'],
+  }),
+  rest: Transformer<TChannel, TChannelGraphql>('rest', {
+    buildFields: ['name', 'description', 'address'],
+  }),
+  graphql: Transformer<TChannel, TChannelGraphql>('graphql', {
+    replaceFields: ({ fields }) => {
+      const name =
+        fields.name &&
+        buildField(fields.name, 'default', {
+          fieldToBuild: 'name',
+        }).en;
+      const nameAllLocales = (fields.name &&
+        buildField(fields.name, 'graphql', {
+          fieldToBuild: 'name',
+        })) as TChannelGraphql['nameAllLocales'];
+
+      return {
+        ...fields,
+        name,
+        nameAllLocales,
+        createdAt: fields.createdAt,
+        createdBy: fields.createdBy as TChannelGraphql['createdBy'],
+        lastModifiedAt: fields.lastModifiedAt,
+        lastModifiedBy:
+          fields.lastModifiedBy as TChannelGraphql['lastModifiedBy'],
```

Comment:
Sorry for the type assertions here for `nameAllLocales`, `createdBy` and `lastModifiedAt`, but I had hard times having it typed and this is the best I could come up with